examples/with-meilisearch/tailwind.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
```

Comment:
Should we use `tailwind.config.ts` here?

Reply:
Moved to TS :)